controllers/apidefinition_controller.go

```diff
@@ -563,10 +669,83 @@ func (r *ApiDefinitionReconciler) updateStatus(
 	return r.Status().Update(ctx, &api)
 }
 
+func (r *ApiDefinitionReconciler) findGraphsForApiDefinition(apiDef client.Object) []reconcile.Request {
```

Comment:
This function will receive subgraph or supergraph resource as arguement. It will not be apiDef.